integration_tests/waypoints.cpp

```diff
@@ -64,276 +37,171 @@ TEST_F(SitlTest, MissionAddWaypointsAndFly)
     Device &device = dc.device();
 
     while (!device.telemetry().health_all_ok()) {
-        std::cout << "waiting for device to be ready" << std::endl;
+        LogInfo() << "waiting for device to be ready";
         std::this_thread::sleep_for(std::chrono::seconds(1));
     }
 
-    std::cout << "Device ready, let's start" << std::endl;
+    LogInfo() << "Device ready, let's start";
 
     std::vector<std::shared_ptr<MissionItem>> mission_items;
 
     mission_items.push_back(
-        add_waypoint(47.398170327054473,
-                     8.5456490218639658,
-                     10.0f, 5.0f, false,
-                     20.0f, 60.0f,
-                     MissionItem::CameraAction::NONE));
+        add_mission_item(47.398170327054473,
```

Comment:
- Is this a takeoff item? 
- Would we be better off having an explicit takeoff and checking for success?

The reason I ask is that this will work for VTOL and MC, but could well fail for Plane, since takeoff is not automatically added for plane missions.

Reply:
Looks like we don't support fixedwing yet then. 😞

Reply:
Fair enough. Should we add a test for vehicle type and exit. Or we could add a takeoff mission item?

Reply:
We could try that but I'm a bit reluctant to change it without more testing in place.